presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2153,6 +2153,20 @@ public void revokeTablePrivileges(ConnectorSession session, SchemaTableName sche
         return result.build();
     }
 
+    @Override
+    public void commitPartitionForCreate(ConnectorSession session, ConnectorOutputTableHandle tableHandle, Collection<Slice> fragments)
```

Comment:
Let's use plural (`commitPartitions`) here, as this method will be used to commit multiple partitions (buckets) at a time.